source/chapters/library.rst

```diff
@@ -840,6 +840,7 @@ iTunes, Traktor, Rhythmbox, Banshee - Using external libraries
 
 .. sectionauthor::
    S.Brandt <s.brandt@mixxx.org>
+   Ukpai Ugochi <outreachy applicant>
 
 Supported libraries:
```

Comment:
Ok! I will do that now!

Reply:
> Serato should be listed here too.
> 
> Also, the title of the section should probably be changed, because if we also add Serato and Rekordbox it becomes too long. How about "Using External Libraries" or "Importing libraries from other software" or something like that?

I have done this! I added Serato to the title, I didn't add RekordBok because nothing was talked about it in that section. The issue for this pull request is **Documenting Imports from Serato**, nothing was said about **RekordBox**. The title of this section already has **Using external libraries** in it. Please let me know if there's any title you think would fit this section more, i would be glad to add it.

Reply:
Thanks. What I meant is that we also support Rekordbox, and if we are going to add that to the section title too it would become a bit long. That's why I suggested to removed all application names from the section and just name it something like "Using libraries from other software".

Also, there is a list "Supported libraries" and Serato is currently missing from that list. Could you add that please? A logo can be found here: https://github.com/mixxxdj/mixxx/blob/master/res/images/library/ic_library_serato.svg

Reply:
> Thanks. What I meant is that we also support Rekordbox, and if we are going to add that to the section title too it would become a bit long. That's why I suggested to removed all application names from the section and just name it something like "Using libraries from other software".
> 
> Also, there is a list "Supported libraries" and Serato is currently missing from that list. Could you add that please? A logo can be found here: https://github.com/mixxxdj/mixxx/blob/master/res/images/library/ic_library_serato.svg

Sure, I would change the title of the section (it certainly does look long). I would also add Serato to the list of supported libraries in Mixxx manual-2.3x. ```"Using libraries from other software"``` the title should fit in, especially if Mixxx supports other software then there would be no need to change title again.

Reply:
> Also, there is a list "Supported libraries" and Serato is currently missing from that list. Could you add that please? A logo can be found here: https://github.com/mixxxdj/mixxx/blob/master/res/images/library/ic_library_serato.svg

Ping.

Reply:
> > Also, there is a list "Supported libraries" and Serato is currently missing from that list. Could you add that please? A logo can be found here: https://github.com/mixxxdj/mixxx/blob/master/res/images/library/ic_library_serato.svg
> 
> Ping.

I am working on this now!

Reply:
> > > Also, there is a list "Supported libraries" and Serato is currently missing from that list. Could you add that please? A logo can be found here: https://github.com/mixxxdj/mixxx/blob/master/res/images/library/ic_library_serato.svg
> > 
> > 
> > Ping.
> 
> I am working on this now!

Hello @Holzhaus I have done this! Please review.